tests/integration/test_auth.py

```diff
@@ -199,24 +127,3 @@ async def test_set_password_action(ops_test: OpsTest):
     assert await validate_sql_queries_with_kyuubi(
         ops_test=ops_test, username=username, password=new_password
     )
-
-
-@pytest.mark.abort_on_fail
-async def test_remove_authentication(ops_test: OpsTest, test_pod, charm_versions):
```

Comment:
I think this test is still relevant here. We can assert the charm goes to blocked state in addition as well.

Reply:
Added in f19f344